src/test/java/org/folio/edge/orders/CommonEndpoint.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.edge.orders;
+
+public enum CommonEndpoint {
+  ORDER_TEMPLATES("/orders/order-templates", "/orders/order-templates", "orderTemplates", true),
+  FUNDS("/finance/funds", "/finance/funds", "funds", true),
+  EXPENSE_CLASSES("/finance/expense-classes", "/finance/expense-classes", "expenseClasses", true),
+  LOCATIONS("/locations", "/locations","locations", true),
+  ACQUISITIONS_UNITS("/acquisitions-units", "/acquisitions-units/units", "acquisitionsUnits", true),
+  ORGANIZATIONS("/organizations", "/organizations/organizations", "organizations", true),
```

Comment:
mod-organizations provides /organizations/organizations, not just /organizations?

Reply:
got it, interesting why mod-orgnization urls are designed in this way